quinn-udp/examples/simple.rs

```diff
@@ -0,0 +1,66 @@
+use anyhow::Result;
+use std::io::IoSliceMut;
+use std::net::Ipv4Addr;
+use std::time::Instant;
+use proto::{EcnCodepoint, Transmit};
```

Comment:
not very happy about this. I guess it would be better to duplicate the definitions and convert it in `quinn`.